EIPS/eip-1576.md

```diff
@@ -0,0 +1,243 @@
+---
+eip: 1576
+title: Simple Whisper bid negotiation protocol
+author: Isaac Patka <isaac+zy@bloom.co>, Dustin van Schouwen <dustin+xy@bloom.co>
+status: Draft
+discussions-to: https://github.com/hellobloom/EIPs/issues/1
+type: Standards Track
+category: ERC
+requires: 627
+created: 2018-07-27
+---
+
+## Simple Summary
+
+Set of protocols for encrypted, dark messaging continuity and bid negotiation on Whisper over ÐΞVp2p/libp2p.
+
+## Abstract
+
+The following standard specifies two protocols - a low-level "bulletin" protocol for publicly posting a message and then dissociating into anonymous Whisper topics for message anonymity/obfuscation, and, on top of that, a protocol for publishing a solicitation for bids that dissociate into encrypted private channels with each message, finalizing, optionally, with a message providing information specific to or required for the on-chain communication of information relating to some good or service.
+
+## Motivation
+
+On-chain negotiation for different providers to offer bids on a given on-chain action is costly, while off-chain negotiation suffers from inability to adequately find peers without centralization.  Whisper protocol built on top of ÐΞVp2p, given adequate peer association, offers several advantages for a negotiation protocol, including anonymity, obfuscation of message origin, and built-in encryption, as well as the added benefit for Ethereum-based systems of reuse of existing technology, as Whisper functionality is built into the standard geth or Parity client.  A decentralized marketplace involving bid solicitation, or any other sort of negotiation process, as a prerequisite to on-chain transactions benefits from all of the above.
+
+## Rationale
+
+A simple, lightweight and fully flexible protocol for messaging chaining, and then additionally for economic negotiations, was desired.  An original protocol of fixed-price-for-good negotiations was replaced with an abstract "bid_type" key and secondary parameters to accomodate any type of economic transaction.  Top-level fields in the protocol were reserved for key protocol features, while the "data" key hosts type-specific information.
+
+# Message Protocol
```

Comment:
These seem to have the wrong size. I think both of these headings should be under `Specification`.